docs/tasks/administer-cluster/opaque-integer-resource-node.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Advertise Opaque Integer Resources for a Node
+---
+
+
+{% capture overview %}
+
+This page shows how to specify opaque integer resources for a Node.
```

Comment:
I want to highlight that this feature is `alpha` and that the API may change or be removed alltogether.